kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/modem/ModemDriver.java

```diff
@@ -281,16 +281,22 @@ public boolean resetModem() {
                 echoSysfsResource(RELIAGATE_10_05_GSM_USB_PATH, true);
 
                 // wait until the modem is on again
-                // isOn uses lsusb that is not supported on the Reliagate 10-05
-                // int cnt = 10;
-                // while (!isOn() && cnt > 0) {
-                // sleep(1000);
-                // cnt--;
-                // }
-                // if (!isOn()) {
-                // retVal = false;
-                // }
-            } catch (Exception e) {
+                int cnt = 10;
```

Comment:
I think that maybe having retVal defined as false, than in line 289 if (isOn) { retVal = true;}
This would prevent having so many retVal=false in the catches.